eZ/Bundle/EzPublishRestBundle/Tests/Functional/SessionTest.php

```diff
@@ -115,7 +115,8 @@ public function testLoginWithExistingFrontendSession()
         $this->setSessionInput($request);
         $request->addHeader("Cookie: $sessionCookie");
         $response = $this->sendHttpRequest($request);
-        self::assertHttpResponseCodeEquals($response, 201);
+        // Since Session is reused, not created, expect 200 instead of 201
+        self::assertHttpResponseCodeEquals($response, 200);
```

Comment:
Note to reviewers: I think this never checked what was supposed to check. [If Session is supposed to be reused, not created, it should return `200`](https://github.com/ezsystems/ezpublish-kernel/blob/v6.7.5/eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/UserSession.php#L29).

Debugging showed me that on 1.x we had `AnonymousToken` there, so session was always created resulting in `201`. The reason for that could be erroneous form action route, which also has been fixed above.

Reply:
So this could be backported then?

Reply:
Yes, occured to me as well, let's check if this works on 6.7, see #2181